pkg/plugins/trigger/generic-comment.go

```diff
@@ -94,6 +98,14 @@ func handleGenericComment(c Client, trigger *plugins.Trigger, gc scmprovider.Gen
 	if err != nil {
 		return err
 	}
+  if arg != "" {
+    for i := range toTest {
+      toTest[i].Base.PipelineRunParams = append(toTest[i].Base.PipelineRunParams , job.PipelineRunParam{
+        Name: "TRIGGER_COMMAND_ARG",
+        ValueTemplate: arg,
+      })
+    }
+  }
```

Comment:
The feature should be documented and unit test added

Reply:
836cdca8f763e195c00a4dedf80fa5020f0ea109 adding in unit tests. please let me know if I have missed anything. I updated the code so that handleGenericComment take in the command arg.
documentation for this feature is in
https://github.com/jenkins-x/lighthouse/blob/1b480280281582ed322d7eaa2131617fd364624a/docs/trigger/README.md?plain=1#L5